simbrain.c

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2007-2017. The SimGrid Team. All rights reserved.          */
+
+/* This program is free software; you can redistribute it and/or modify it
+ * under the terms of the license (GNU LGPL) which comes with this package. */
+
+#include "simgrid/msg.h"
+#include <math.h>
+#include <unistd.h>
+
+XBT_LOG_NEW_DEFAULT_CATEGORY(SimBrain, "Messages specific for this msg example");
+
+static int output(int argc, char *argv[])
+{
+  xbt_assert(argc==2, "The output function one argument from the XML deployment file");
+  
+
+  double b0 = 1; 
+  double mp = 1;
+  double p;
```

Comment:
Add comments to explain what these variables are. `b0` and `mp` should be parameters of the simulation rather than hard-coded values.